components/net/filemanager_thread.rs

```diff
@@ -440,8 +441,13 @@ impl FileManagerStore {
                     bytes: buf.bytes.index(range).to_vec(),
                 };
 
-                let _ = sender.send(Ok(ReadFileProgress::Meta(buf)));
-                let _ = sender.send(Ok(ReadFileProgress::EOF));
+                println!("Sending bytes: {:?}", buf);
```

Comment:
Let's remove the uses of println! in this PR. If we still want the debugging output in the final code, trace! or debug! would be more appropriate.